libass/ass_fontconfig.c

```diff
@@ -362,7 +372,8 @@ ass_fontconfig_add_provider(ASS_Library *lib, ASS_FontSelector *selector,
     provider = ass_font_provider_new(selector, &fontconfig_callbacks, fc);
 
     // build database from system fonts
-    scan_fonts(fc->config, provider);
+    if (!scan_fonts(fc->config, provider))
+        ass_msg(lib, MSGL_FATAL, "Failed to load fonctconfig fonts!");
```

Comment:
Not a blocker:

This matches the level of the “No valid fontconfig configuration found!” message, but perhaps both should be `MSGL_ERROR`, seeing as libass continues to function, just without system fonts?

Reply:
done